hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -601,7 +601,7 @@ ppFailureReport name (FailureReport size seed _ inputs0 mlocation0 msg mdiff msg
         WL.vsep . WL.punctuate WL.line
     , with decls $
         WL.vsep . WL.punctuate WL.line . fmap ppDeclaration
-    , with msgs $
+    , with msgs
```

Comment:
Yeah, I thought the same on that one but wasn't really sure. I will put it back 👍